app/models/timeline_event.rb

```diff
@@ -66,7 +66,7 @@ def self.valid_checklist_status
   attr_accessor :files
 
   def reviewed?
-    timeline_event_grades.present?
+    evaluated_at.present? or passed_at.present?
```

Comment:
I think we just need to look at the `evaluated_at`
```suggestion
    evaluated_at.present?
```